stdlib/public/SwiftShims/GlobalObjects.h

```diff
@@ -45,6 +45,9 @@ __swift_uint64_t _swift_stdlib_HashingDetail_fixedSeedOverride;
 extern SWIFT_RUNTIME_STDLIB_INTERFACE
 void *_swift_stdlib_ProcessArguments;
 
+extern SWIFT_RUNTIME_STDLIB_INTERFACE
+void *_swift_stdlib_ProcessArgv;
```

Comment:
I think it should be called `_swift_stdlib_ProcessUnsafeArgv` to match the property name.